react-toggle-switch/my-app/src/ToggleSwitch.tsx

```diff
@@ -0,0 +1,34 @@
+import './ToggleSwitch.css';
+import { useState } from 'react';
+
+export function ToggleSwitch() {
+  const [isOn, setSwitch] = useState(false);
+  let container: string = '';
+  let active: string = '';
+  let label: string = '';
+
+  if (isOn === true) {
```

Comment:
Since `isOn` is a boolean, it's considered better code to just write `if (isOn)`.